detox/src/devices/drivers/ios/SimulatorDriver.js

```diff
@@ -215,12 +215,8 @@ class SimulatorDriver extends IosDriver {
 
   async _groupDevicesByStatus(deviceQuery) {
     const searchResults = await this._queryDevices(deviceQuery);
-
-    const { busy, free}  = _.groupBy(searchResults, device => {
-      return this.deviceRegistry.isDeviceBusy(device.udid)
-        ? 'busy'
-        : 'free';
-    });
+    const takenDevices = this.deviceRegistry.getRegisteredDevices();
+    const { busy, free }  = _.groupBy(searchResults, ({ udid }) => takenDevices.includes(udid) ? 'busy' : 'free');
```

Comment:
why not rename it to tekken and not-tekken ? 😀

Reply:
See #2573